staNMF/staNMF.py

```diff
@@ -134,16 +133,16 @@ def load_data(self):
             try:
                 csvfile = open(self.fn, "r")
             except:
-                ("File ''" + str(self.fn) + "' not found.")
+                print("File ''" + str(self.fn) + "' not found.")
```

Comment:
Shouldn't `self.fn` always be a str, i.e. no need to explicitly call str?

Also I'm not sure the try/except is needed here. Isn't the default error for file not found pretty informative? There isn't much functional difference between your `sys.exit()` failure and the builtin failure.